src/ibrowse_http_client.erl

```diff
@@ -1870,7 +1870,7 @@ cancel_timer(Ref, {eat_message, Msg}) ->
     end.
 
 make_req_id() ->
-    now().
+    {erlang:unique_integer([positive, monotonic]), 0, 0}.
```

Comment:
I saw at least once request ID is sent in a [header](https://github.com/apache/couchdb-ibrowse/blob/3e0e5fffdb63f79d7a55ea38d384fdf4fbede6cf/src/ibrowse_http_client.erl#L970) and could potentially be logged.  And since unique integer counter is reset to 0 on every VM restart the behavior might change. So I think we could use the same scheme as for the filename (pull it into a helper function) and could re-use the same logic.

This might still introduce a minor incompatibility if users ever introspected the value of `ReqId` and expected it to be a proper timestamp. It doesn't seem like ibrowse documents `ReqId` to be a `timestamp()` and the type spec in `browse.erl` just says it's a generic [term()](https://github.com/apache/couchdb-ibrowse/blob/master/src/ibrowse.erl#L162). But if you feel like could add it as a comment somewhere in the code or just in the commit message.